great_expectations/expectations/metrics/query_metric_provider.py

```diff
@@ -0,0 +1,9 @@
+import logging
+
+from great_expectations.expectations.metrics.metric_provider import MetricProvider
+
+logger = logging.getLogger(__name__)
+
+
+class QueryMetricProvider(MetricProvider):
```

Comment:
There should be a docstring explaining the reason this class exists.

Reply:
Will do! Any thoughts on language around the purpose of `MetricProvider`s in general? Happy to update each of them w/ docstrings as currently none have one.